test/JSON/stackoverflow.baseline

```diff
@@ -0,0 +1 @@
+Error: Out of stack space

```

Comment:
nit: could be nice to ditch `baseline file` and compare output and `console.log('PASS')` ?